tests/topotests/kernel_bypass_zebra_ecmp/test_kernel_bypass_ecmp.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+
+#
```

Comment:
Drop all the text and leave only Copyright (your) + SPDX-License-Identifier is a MUST (See other examples).